### PR TITLE
Earn: Remove Badge by Recurring Payments

### PR DESCRIPTION
**Pull request by Aurorum**
#### Changes proposed in this Pull Request

This just simply removes the "New" badge.

#38483, didn't cover this, but let me know if it should stay and the approach in #38440 should be used instead. 

#### Testing instructions

Visit the Earn page and verify the badge next to "Collect recurring payments" is gone 

Screenshot 2019-12-19 at 18 15 23

cc @davemart-in

### CODE REVIEW

**Comment by gwwar** [comment]
https://github.com/Automattic/wp-calypso/pull/38483#issuecomment-567625729

> I guess my take would be that the New badge on the Earn page can stay for all users for a while. My guess is that we'll be adding more options to the Earn page in Q1, at which point we can remove the New badge from Recurring Payments card to the new features that are added.

**Review by gwwar — APPROVED** [review]
Thanks @Aurorum this tests well for me, but I'll leave it up to @artpi or @davemart-in on when to land this.

| Before | After |
| ------------- | ------------- |
| Screen Shot 2019-12-20 at 2 19 20 PM | Screen Shot 2019-12-20 at 2 20 35 PM |

**Comment by davemart-in** [comment]
Cool. Thanks for doing this @Aurorum. I'd say let's hold off on merging this one for now. I left some thoughts regarding this [here](https://github.com/Automattic/wp-calypso/pull/38483).

> I guess my take would be that the New badge on the Earn page can stay for all users for a while. My guess is that we'll be adding more options to the Earn page in Q1, at which point we can remove the New badge from Recurring Payments card to the new features that are added.

**Comment by sixhours** [comment]
It looks like this was removed in a different PR so I'm closing this out. :)

**Pull request closed without merging**
